docs/source/native/intro.rst

```diff
@@ -50,3 +50,25 @@ Variable types
   | You have to create new slots with the ``<-`` operator and overwrite values in slots with ``=``. Tables are explained in more detail in the `squirrel documentation <http://www.squirrel-lang.org/squirreldoc/reference/language/tables.html>`_
 * ``struct``: a series of attributes assigned to a structure and accessed through with ``structname.variablename``
 * ``var``: Var typically refers to variables that lack any stated type, this will cause issues with many functions unless you use the ``untyped`` identifier at the start of your file, or if you use ``expect type(variable)`` when passing the variable to that function
+
+Content types
+-----
+
+tables and arrays can have their content types specified. If none are given, they will default to ``var``.
+
+For tables you can set the types of their keys and contents.
+
+.. code-block:: javascript
+
+    // table<key_type, value_type> identifier
+    table<string, entity> players
+    foreach( entity player in GetPlayerArray() ) // insert every player into the table indexable by their usernames
+    {
+        players[ player.GetPlayerName() ] <- player
+    }
```

Comment:
Would maybe recommend adding an example of nested tables and nested arrays too?

Reply:
idk seemd intuitive, doesn't it? The only confusing thing is that the bit right shift operator is being recognized before content params

Reply:
well then add something about that maybe :P